indexer/cli/cli.go

```diff
@@ -99,6 +99,7 @@ func (c *Cli) Run(args []string) error {
 
 func NewCli(GitVersion string, GitCommit string, GitDate string) *Cli {
 	flags := []cli.Flag{ConfigFlag}
+	flags = append(flags, log.CLIFlags("indexer")...)
```

Comment:
before merging, can you verify that the prefix here ends up upper-cased in the end? i.e `INDEXER_LOG_LEVEL`

Reply:
```suggestion
	flags = append(flags, log.CLIFlags("INDEXER")...)
```